Classes/Backend/SolrModule/IndexQueueModuleController.php

```diff
@@ -67,7 +67,7 @@ public function indexAction() {
 	 * @return void
 	 */
 	public function initializeIndexQueueAction() {
-		$initializedIndexingConfigurations = array();
+		$initializedIndexingConfigurations = [];
```

Comment:
Please revert these unrelated changes. Also, array short syntax is only available with PHP 5.4+, but CMS 6.2 only requires a minimum version of 5.3.7.